<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerBeamsModule</class>
 <widget class="qSlicerWidget" name="qSlicerBeamsModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Beams</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="10" column="0">
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_BeamParameters">
     <property name="text">
      <string>Beam parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="label_BeamName">
          <property name="text">
           <string>Beam name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_BeamName"/>
        </item>
        <item>
         <widget class="QLabel" name="label_beamWeight">
          <property name="text">
           <string>Beam weight:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_BeamWeight">
          <property name="minimumSize">
           <size>
            <width>64</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="qMRMLBeamParametersTabWidget" name="BeamParametersTabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_RtBeam">
       <property name="nodeTypes">
        <stringlist>
         <string>vtkMRMLRTBeamNode</string>
        </stringlist>
       </property>
       <property name="noneEnabled">
        <bool>true</bool>
       </property>
       <property name="addEnabled">
        <bool>false</bool>
       </property>
       <property name="renameEnabled">
        <bool>true</bool>
       </property>
       <property name="noneDisplay">
        <string>Beams can be created under plans in External Beam Planning</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Active RT beam:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_SwitchToParentPlan">
       <property name="minimumSize">
        <size>
         <width>144</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Switch to parent plan</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/Icons/Plan.png</normaloff>:/Icons/Plan.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLBeamParametersTabWidget</class>
   <extends>QTabWidget</extends>
   <header>qMRMLBeamParametersTabWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../SubjectHierarchyPlugins/Resources/qSlicerBeamsSubjectHierarchyPlugins.qrc"/>
  <include location="../../../DicomRtImportExport/SubjectHierarchyPlugins/Resources/qSlicerDicomRtImportExportSubjectHierarchyPlugins.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>qSlicerBeamsModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLNodeComboBox_RtBeam</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MRMLNodeComboBox_RtBeam</sender>
   <signal>currentNodeChanged(vtkMRMLNode*)</signal>
   <receiver>BeamParametersTabWidget</receiver>
   <slot>setBeamNode(vtkMRMLNode*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
